spec/std/isa/inst/Q/fmaxm.q.yaml

```
@@ -6,9 +6,17 @@
 $schema: "inst_schema.json#"
 kind: instruction
 name: fmaxm.q
-long_name: No synopsis available
-description: |
-  No description available.
+long_name: Floating-Point Maximum-Number with _NaN_ Handling Quad-Precision
```

`fmax` handles NaN, too, just differently. I'm not sure if the `m` at the end of the mnemonic has a specific meaning. Let's just put "NaN" here for now, since this instruction prefers NaN over non-Nan:
```suggestion
long_name: Floating-Point Maximum-Number NaN Quad-Precision
```